### PR TITLE
Expose the lender name.

### PR DESCRIPTION
**Pull request by nickcharlton**
See: https://trello.com/c/NmzoqmQt

* [x] Loan Portfolio
* [x] Loan Show

Best example loan with a `sub_lender`: 126939

### CODE REVIEW

File: app/views/loan_states/show.html.erb [hunk 1]
```diff
@@ -22,6 +23,7 @@
     <% @loans.each do |loan| %>
       <tr>
         <td><%= link_to(loan.reference, loan) %></td>
+        <%= "<td>#{loan.lender.name}</td>".html_safe unless current_user.is_a?(LenderUser) %>
```
Comment (tjmw) [inline app/views/loan_states/show.html.erb]: Personal pref but this looks a bit more natural to me:

```erb
<% unless current_user.is_a?(LenderUser) %>
  <td><%= loan.lender.name %></td>
<% end %>
```

What do you think? I guess I like not having to call `html_safe` if I can avoid it.

Comment (nickcharlton) [inline app/views/loan_states/show.html.erb]: Ooh, that's nicer.

File: app/views/loan_states/show.html.erb [hunk 2]
```diff
@@ -13,6 +13,7 @@
   <thead>
     <tr>
       <th>Reference</th>
+      <%= "<th>Lender</th>".html_safe unless current_user.is_a?(LenderUser) %>
```
Comment (sidane) [inline app/views/loan_states/show.html.erb]: You can use the `content_tag` helper for this:

`content_tag(:th, "Lender") unless current_user.is_a?(LenderUser)`
`content_tag(:td, loan.lender.name) unless current_user.is_a?(LenderUser)`

**Comment by tjmw** [comment]
:+1: LGTM, with that one minor comment to take or leave.

**Comment by sidane** [comment]
What Tom said 😄

File: spec/requests/loan_states_spec.rb [hunk 3]
```diff
@@ -20,100 +22,147 @@ def counts(state)
...
 
-      expect(page.current_url).to eq(loan_state_url(id: 'completed', format: 'csv'))
+        expect(page).not_to have_text("Lender")
+        expect(page).not_to have_text(current_lender.name)
+      end
     end
 
-    it 'exports loan data as a CSV with a scheme filter' do
-      dispatch(id: 'completed', scheme: 'efg')
+    context "as any other user" do
```
Comment (sidane) [inline spec/requests/loan_states_spec.rb]: Worth adding a test here to ensure sub lender name is used when present?

I think you can probably add `sub_lender: "whatever"` to one of the loans created in the before block.

**Comment by nickcharlton** [comment]
Added an additional test for the `sub_lender`. Now merging.

**Pull request merged**